string command
float64[] data
---
bool success
